v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/options/OptionsToConfigBuilder.java

```diff
@@ -236,9 +237,10 @@ private static JdbcIOWrapperConfig.Builder builderWithDefaultsFor(SQLDialect dia
     return builderWithMySqlDefaults();
   }
 
+  // TODO(vardhanvthigle): Standardize for Css.
   private static SourceSchemaReference sourceSchemaReferenceFrom(
       SQLDialect dialect, String dbName, String namespace) {
-    SourceSchemaReference.Builder builder = SourceSchemaReference.builder();
+    JdbcSchemaReference.Builder builder = JdbcSchemaReference.builder();
```

Comment:
please add a switch case or so here where it checks for a jdbc source. The other flows should throw an unsupported exception.

Reply:
The code is not checking for JDBC here. We are constructing a JDBC schema reference  here.
As of now the config only constructs JDBC, as we don't have any support for Cassandra as yet.

The Todo on top of this function is to add support for config with which the user could connect to Cassandra but that will come in later PRs.